sklearn/tree/export.py

```diff
@@ -14,7 +14,7 @@
 
 from ..externals import six
 
-from . import _tree
+from . import _tree, _criterion
```

Comment:
This is better to have one import per line as it's easier to see changes whenever you use a software like git.